flask_api.py

```diff
@@ -108,8 +109,14 @@ def handle_question():
     if "question" not in request_body:
         return "request body should include the question", BAD_REQUEST
 
+    # Store
+    initializeDB()
+    try:
```

Comment:
I haven't done any testing, but consider only inserting a QuestionLog when "question" is not None (when a "question" field is found in the request).